### DISCUSSION TITLE
新版蓝灯在Mac上乱码，无法付款购买

### DISCUSSION THREAD

**Discussion by skar9363** [getlantern/lantern-forum-cn#506] [Q&A]
前两天蓝灯到期，想要续费时提示我要更新到最新版本，更新之后发现界面都变了，而且打开之后全是乱码，输入邮箱也都是乱码，无法付费购买，可否解决一下？
PS：最新稳定版和测试版我都试了，都是乱码（准确的说是英文乱码，中文正常）

![CleanShot 2024-05-13 at 15 08 11](https://github.com/getlantern/lantern-forum-cn/assets/864313/825ccc68-980f-49bd-98d6-f068befc9661)
![CleanShot 2024-05-13 at 15 08 01](https://github.com/getlantern/lantern-forum-cn/assets/864313/34e1582b-dcca-41d7-be86-2d8d29e2d377)

**Comment by MaxMKMax** [comment]
@skar9363 嗯嗯. 谢谢您的反馈. 我们马上修复. 谢谢您让我们知道. 您明天或后天再回来购买. 不好意思啊! 谢谢您支持蓝灯!
